content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md

```diff
@@ -74,12 +74,15 @@ aliases:
 - **Added** values to the Istio Gateway Helm charts for configuring annotations on the ServiceAccount.  Can be used to enable [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) on AWS EKS.
   ([Pull Request #33914](https://github.com/istio/istio/pull/33914))
 
-- **Added** support for JWT claim based routing on ingress gateways.
+- **Added** support for [JWT claim based routing](/docs/tasks/security/authentication/jwt-route) on ingress gateways.
   ([Pull Request #35762](https://github.com/istio/istio/pull/35762))
 
 - **Fixed** the `EnvoyExternalAuthorizationHttpProvider` to match HTTP headers in a case-insensitive way.
   ([Issue #35220](https://github.com/istio/istio/issues/35220))
 
+- **Graduated** the [external authorization](/docs/tasks/security/authorization/authz-custom) from experimental to alpha stage.
```

Comment:
Graduated is not a valid action: https://github.com/istio/istio/tree/master/releasenotes#release-notes. Promoted would be appropriate.

Reply:
fixed, thanks for the link.